rest-api-spec/src/main/resources/rest-api-spec/api/indices.get_sample.json

```diff
@@ -0,0 +1,38 @@
+{
+  "indices.get_sample": {
+    "documentation": {
+      "url": "https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-get-sample",
+      "description": "Get random sample of ingested data"
+    },
+    "stability": "experimental",
+    "visibility": "public",
+    "headers": {
+      "accept": [
+        "application/json"
+      ]
+    },
+    "url": {
+      "paths": [
+        {
+          "path": "/{index}/_sample",
+          "methods": [
+            "GET"
+          ],
+          "parts": {
+            "index": {
+              "type": "string",
+              "description": "The name of a data stream or index"
+            }
+          }
+        }
+      ]
+    },
+    "params": {
+      "master_timeout": {
+        "type": "time",
+        "default": "30s",
+        "description": "Specify timeout for connection to master"
```

Comment:
AFAIK, you don't actually support this parameter, so it shouldn't be in the spec either.

Reply:
Oops, yeah copy/paste mistake.